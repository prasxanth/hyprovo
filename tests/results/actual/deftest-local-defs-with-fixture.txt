2021-11-24 22:51:24.643 - INFO - Pass ✓... (deftest-local-defs-with-fixture / test-add-variable-local-defs): add pass
2021-11-24 22:51:24.645 - ERROR - Fail ✗... (deftest-local-defs-with-fixture / test-add-variable-local-defs): add fail
2021-11-24 22:51:24.649 - INFO - Pass ✓... (deftest-local-defs-with-fixture / test-div-variable): ratio should be 1
2021-11-24 22:51:24.650 - ERROR - Fail ✗... (deftest-local-defs-with-fixture / test-div-variable): ratio should be 4
2021-11-24 22:51:24.650 - DEBUG - (deftest-local-defs-with-fixture): Test Summary → Total: 4    Pass: 2    Fail: 2
