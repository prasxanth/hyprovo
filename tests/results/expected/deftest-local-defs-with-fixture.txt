2021-11-24 22:50:31.490 - INFO - Pass ✓... (deftest-local-defs-with-fixture / test-add-variable-local-defs): add pass
2021-11-24 22:50:31.492 - ERROR - Fail ✗... (deftest-local-defs-with-fixture / test-add-variable-local-defs): add fail
2021-11-24 22:50:31.497 - INFO - Pass ✓... (deftest-local-defs-with-fixture / test-div-variable): ratio should be 1
2021-11-24 22:50:31.499 - ERROR - Fail ✗... (deftest-local-defs-with-fixture / test-div-variable): ratio should be 4
2021-11-24 22:50:31.499 - DEBUG - (deftest-local-defs-with-fixture): Test Summary → Total: 4    Pass: 2    Fail: 2
